local.conf.appendix

```diff
@@ -1,3 +1,6 @@
+# Needed since a bbappend in meta-ivi adds libusb PACKAGECONFIG which is
+# broken for some weird reason
+BBMASK .= "|meta-ivi/recipes-devtools/qemu/"
```

Comment:
Did we report a bug?

Reply:
Not yet. I will discuss this with @frznlogic and see if we can find a way forward (either we find the solution or we just report it as a bug).

Reply:
I have talked to @frznlogic and we will look into this. It will not be done within this PR though.

Reply:
Sure, just so we report the bug at least.

Reply:
sorry, I missed this discussion earlier. I did look at it briefly, the PACKAGECONFIG is super old... I'll remove it and see what people/the build system think of it.